packages/trader/src/Modules/Trading/Components/Elements/purchase-button.tsx

```diff
@@ -119,8 +119,8 @@ const PurchaseButton = ({
                 'btn-purchase--multiplier': is_multiplier,
                 'btn-purchase--multiplier-deal-cancel': has_deal_cancellation,
                 'btn-purchase--turbos': is_turbos,
-                'btn-purchase--1__vanilla-opts': index === 0 && is_vanilla,
-                'btn-purchase--2__vanilla-opts': index === 1 && is_vanilla,
+                'btn-purchase--has-bottom-gradient-green': index === 0 && (is_accumulator || is_vanilla || is_turbos),
```

Comment:
Moved common style (gradient for purchase button) into a separate className `btn-purchase--has-bottom-gradient`. Previously it was duplicated for Accu and Vanillas. Added sanity to QA-checklist